esp/esp/program/models/class_.py

```diff
@@ -1111,7 +1115,27 @@ def firstBlockEvent(self):
     def isFull(self, ignore_changes=False, webapp=False):
         if len(self.get_meeting_times()) == 0:
             return True
-        if webapp and Tag.getBooleanTag('count_checked_in_only', program = self.parent_program, default = False):
+
+        now = datetime.datetime.now()
+        switch_time = None
+        if Tag.getProgramTag('switch_time_program_attendance', program=self.parent_program):
+            try:
+                switch_hour, switch_min = [int(x) for x in Tag.getProgramTag('switch_time_program_attendance', program=self.parent_program).split(":")]
+                switch_time = datetime.datetime(now.year, now.month, now.day, switch_hour, switch_min)
+            # TODO: Is this the best way to deal with someone specifying the tag incorrectly?
+            except ValueError:
```

Comment:
If you make the change I suggested above, you'll want to catch whatever error would be thrown for an incorrectly formatted time. But otherwise this looks like a nice way to catch incorrectly specified tag values.